N2A/src/gov/sandia/n2a/ui/settings/SettingsRepo.java

```diff
@@ -1143,8 +1135,9 @@ public void create (int row, String URL)
             GitWrapper gitRepo = new GitWrapper (baseDir.resolve (".git"));
             gitRepo.setURL (URL);
             AppData.repos.set (1, name, "visible");  // Implicitly creates the repo node.
-            existingModels    .put (name, new MDir (name, baseDir.resolve ("models")));
-            existingReferences.put (name, new MDir (name, baseDir.resolve ("references")));
+            // Local variable temp must be final or effectively final to be used in the following forEach.
```

Comment:
Instead of a lambda expression, you could iterate over an entry set (existing.entrySet()). In any case, the fact that a lambda requires final values is not worth commenting (except to gripe about Java's overly-strict rules about effective-final).